tools/csv-merger/csv-merger.go

```diff
@@ -619,7 +619,7 @@ The NooBaa operator deploys and manages the [NooBaa][2] Multi-Cloud Gateway on O
 	ocsCSV.Annotations["features.ocs.openshift.io/enabled"] = `["external"]`
 	// Used by UI to validate user uploaded metdata
 	// Metadata is used to connect to an external cluster
-	ocsCSV.Annotations["external.features.ocs.openshift.io/validation"] = `{"secrets":["rook-ceph-operator-creds", "rook-csi-rbd-node", "rook-csi-rbd-provisioner", "rook-csi-cephfs-node", rook-csi-cephfs-provisioner"], "configMaps": ["rook-ceph-mon-endpoints", "rook-ceph-mon"], "storageClasses": ["rook-ceph-retain-bucket"]}`
+	ocsCSV.Annotations["external.features.ocs.openshift.io/validation"] = `{"secrets":["rook-ceph-operator-creds", "rook-csi-rbd-node", "rook-csi-rbd-provisioner", "rook-csi-cephfs-node", "rook-csi-cephfs-provisioner"], "configMaps": ["rook-ceph-mon-endpoints", "rook-ceph-mon"], "storageClasses": ["rook-ceph-retain-bucket", "ceph-rgw", "ceph-rbd", "cephfs"]}`
```

Comment:
Shouldn't you get rid of `rook-ceph-retain-bucket`?

Reply:
yes...